src/python_testing/TC_RVCOPSTATE_2_4.py

```diff
@@ -123,55 +104,62 @@ async def test_TC_RVCOPSTATE_2_4(self):
         rvc_op_states = Clusters.RvcOperationalState.Enums.OperationalStateEnum
         op_errors = Clusters.OperationalState.Enums.ErrorStateEnum
 
-        self.print_step(1, "Commissioning, already done")
-
-        if self.check_pics("RVCOPSTATE.S.M.ST_STOPPED"):
-            self.print_instruction(2, "Manually put the device in the STOPPED operational state")
-
-            await self.read_operational_state_with_check(3, op_states.kStopped)
-
-            await self.send_go_home_cmd_with_check(4, op_errors.kNoError)
-
-            await self.read_operational_state_with_check(5, rvc_op_states.kSeekingCharger)
+        # These are the mode values used by the RVC example app that is used in CI.
+        rvc_app_run_mode_idle = 0
+        rvc_app_run_mode_cleaning = 1
```

Comment:
I would move these two next to the code where they are used (line 154).

Reply:
I know that currently these are only used in one place, but if they weren't, this is where I would put them. Given the possibility of changes in the future I think that this is a good place for them.